Tty.xs

```diff
@@ -186,11 +186,11 @@ mysignal(int sig, mysig_t act)
  * will be copied.  Always NUL terminates (unless siz == 0).
  * Returns strlen(src); if retval >= siz, truncation occurred.
  */
-static size_t
-strlcpy(dst, src, siz)
-        char *dst;
-        const char *src;
-        size_t siz;
+size_t
+strlcpy(
+        char *dst,
+        const char *src,
+        size_t siz)
 {
```

Comment:
definition from `musl` library https://git.musl-libc.org/cgit/musl/tree/include/string.h#n86
note that `HAVE_STRLCPY` is not set... so this is why we have a duplicate definition 
would not checking for `_STRING_H` or `__HAVE_ARCH_STRLCPY` be a better alternate?

Reply:
we should probably not redefine this function when not needed